tools/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "rav1d_cli"
+authors = ["C2Rust"]
+version = "0.2.0"
+publish = false
+edition = "2021"
+default-run = "dav1d"
+
+[[bin]]
+path = "dav1d.rs"
```

Comment:
Can we put them in the normal location now?  `src/bin/dav1d.rs` I think.

Reply:
We don't need to right now, no.